packages/now-next/src/index.ts

```diff
@@ -70,6 +70,7 @@ import {
   syncEnvVars,
   validateEntrypoint,
 } from './utils';
+import findUp from 'find-up';
```

Comment:
Note that you can use the `walkParentDirs()` function from `@vercel/build-utils` which I think does the same thing.

Reply:
It looks like the `walkParentDirs` method only handles looking up one file right now and we're looking for either `vercel.json` or `now.json` so it might be better to keep using `find-up` instead of modifying the existing method for now